programs/voter-stake-registry/tests/program_test/governance.rs

```diff
@@ -166,14 +176,18 @@ impl GovernanceRealmCookie {
                 &authority.pubkey(),
                 Some(voter.voter_weight_record),
                 spl_governance::state::governance::GovernanceConfig {
-                    vote_threshold_percentage:
-                        spl_governance::state::enums::VoteThresholdPercentage::YesVote(50),
-                    min_community_weight_to_create_proposal: 1000,
+                    min_community_weight_to_create_proposal: 1,
```

Comment:
why did this value change?

Reply:
it was an arbitrary value copied from new version of the tests. I set it back to 1000.